.gitignore

```diff
@@ -9,10 +9,4 @@ lib-cov
 
 pids
 logs
-results
-
-npm-debug.log
-node_modules/*
-package-lock.json
```

Comment:
Let's revert these changes in `.gitignore` file.

Reply:
I've tried to do this, but I'm not sure I've done it correctly. Please let me know if there's still an issue.